projects/soft_999_shicheng_lu_fw_102/src/main.c

```diff
@@ -0,0 +1,39 @@
+#include <stdint.h>
+
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define DELAY_MS 500
+
+typedef struct Counters {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+
+void increment_counters(SoftTimerId id, void *context) {
```

Comment:
This function should be static and you should append a prv_ prefix in the name of the function to indicate it is private.

Eg
`prv_increment_counters_callback`